### ISSUE TITLE
Sitemap connection issue

### ISSUE THREAD

**Issue by advisionmarketing** [monstra-cms/monstra#409]
Hey all,

I'd love your feedback on the sitemap for one of our clients. Visiting [http://www.5280exteriors.com/sitemap.xml](http://www.5280exteriors.com/sitemap.xml) requires the visitor to login, which is preventing us from submitting the sitemap to search engines. Any suggestions for how to make this file publicly visible?

thanks in advance!
pat

**Comment by mightyhaggis** [comment]
@advisionmarketing this does not look like a Monstra issue, but rather an issue with you hosting set-up.

Browsing to the link provided and Cancelling gives a Windows IIS error: 
`HTTP Error 401.3 - Unauthorized: Access is denied due to an ACL set on the requested resource.`
Google-ing this suggests its a directory permission error.

**Comment by advisionmarketing** [comment]
@mightyhaggis you're absolutely right! I just can't figure out how to fix this to update the permission settings in FTP

**Comment by mightyhaggis** [comment]
@advisionmarketing I tend to use LAMP based hosting rather than windows so can't really advise.
But assuming you have tried changing folder permissions in FTP and its not worked you might need to speak to who ever runs your web server.

**Comment by ghost** [comment]
You can't fix the permissions in FTP, you need to manage the server from IIS Manager, which means you need direct access to the server. If it's not your server you might be able to submit a support ticket to ask the provider to allow access to the xml file.

You seem to have disabled the sitemap url plugin also, since the /sitemap url doesn't work. As long as all your pages are linked within your website the search-bot crawlers should still manage to index your site.
